### PR TITLE
Fixing sample for tests

### PR DESCRIPTION
**Pull request by BrennanConroy**
@Tratcher @ryanbrandenburg @pranavkm

### CODE REVIEW

**Comment by Tratcher** [comment]
Do they still need publish? https://github.com/aspnet/Localization/blob/dev/test/Microsoft.AspNetCore.Localization.FunctionalTests/TestRunner.cs#L40

**Comment by ryanbrandenburg** [comment]
:shipit: from what I can tell, but you should probably wait for @Tratcher as I think he knows more about the problems the area was having.

**Comment by BrennanConroy** [comment]
Changed @Tratcher

File: test/Microsoft.AspNetCore.Localization.FunctionalTests/TestRunner.cs [hunk 1]
```diff
@@ -37,7 +37,7 @@ public TestRunner(string applicationPath)
                 {
                     ApplicationBaseUriHint = applicationBaseUrl,
                     EnvironmentName = environmentName,
-                    PublishApplicationBeforeDeployment = true,
+                    PublishApplicationBeforeDeployment = false,
```
Comment (muratg) [inline test/Microsoft.AspNetCore.Localization.FunctionalTests/TestRunner.cs]: Why this change?

**Comment by muratg** [comment]
Looks good to me, though @Tratcher should give the squirrel :)

File: samples/LocalizationSample/project.json [hunk 2]
```diff
@@ -4,33 +4,23 @@
     "Microsoft.AspNetCore.IISPlatformHandler": "1.0.0-*",
     "Microsoft.AspNetCore.Localization": "1.0.0-*",
     "Microsoft.Extensions.Localization": "1.0.0-*",
-    "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*",
-    "Microsoft.NETCore.Platforms": "1.0.1-*"
```
Comment (pranavkm) [inline samples/LocalizationSample/project.json]: Apparently you need this to make Kestrel work in OSX.

Comment (pranavkm) [inline samples/LocalizationSample/project.json]: cc @JunTaoLuo

File: test/Microsoft.AspNetCore.Localization.FunctionalTests/TestRunner.cs (hunk 1 above)
Comment (Tratcher) [inline test/Microsoft.AspNetCore.Localization.FunctionalTests/TestRunner.cs]: He's reverting my prior workaround.

Comment (Tratcher) [inline test/Microsoft.AspNetCore.Localization.FunctionalTests/TestRunner.cs]: You can just remove this line.

File: samples/LocalizationSample/project.json (hunk 2 above)
Comment (JunTaoLuo) [inline samples/LocalizationSample/project.json]: Yes keep NETCore.Platforms. I'm working to add it to all apps.

**Comment by BrennanConroy** [comment]
:up: :date:

File: samples/LocalizationSample/project.json [hunk 3]
```diff
@@ -10,27 +10,18 @@
   "compilationOptions": {
     "emitEntryPoint": true
   },
-  "commands": {
-    "web": "LocalizationSample"
-  },
+  "content": [
+    "wwwroot/**/*"
```
Comment (JunTaoLuo) [inline samples/LocalizationSample/project.json]: isn't `"wwwroot"` sufficient? I don't think the globbing is necessary.

Comment (BrennanConroy) [inline samples/LocalizationSample/project.json]: Fixed

**Comment by JunTaoLuo** [comment]
:shipit:
